src/features/mod-activity.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  Client,
+  AuditLogEvent,
+  GuildMember,
+  PartialGuildMember,
+} from "discord.js";
+import { logger } from "./log";
+
+const guildMemberTimeoutHandler = (
+  oldMember: GuildMember | PartialGuildMember,
+  newMember: GuildMember,
+) => {
+  const oldTimeout = oldMember.communicationDisabledUntil;
+  const newTimeout = newMember.communicationDisabledUntil;
+
+  if (!oldTimeout && newTimeout) {
+    logger.log(
```

Comment:
Oh – these may make more sense to put into #mod-log, but the existing helpers for doing so are a little heavier-weight than just logging. It's got different types of logs defined as constants and a function to render them appropriately, which might not make sense in this context

Reply:
Sure thing! The previous PR enables just that - they now report to the `modLog`